src/fdc3/dotnet/DesktopAgent/test/MorganStanley.ComposeUI.DesktopAgent.Tests/EndToEndTests.cs

```diff
@@ -23,48 +22,45 @@
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Infrastructure.Internal;
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Tests.Helpers;
 using MorganStanley.ComposeUI.Messaging.Client.WebSocket;
-using MorganStanley.ComposeUI.Messaging.Abstractions;
+using MorganStanley.ComposeUI.MessagingAdapter.Abstractions;
 using MorganStanley.ComposeUI.ModuleLoader;
+using static MorganStanley.ComposeUI.Fdc3.DesktopAgent.Tests.TestData.TestAppDirectoryData;
 using AppIdentifier = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.AppIdentifier;
 using AppIntent = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.AppIntent;
 using AppMetadata = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.AppMetadata;
-using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol;
 using DisplayMetadata = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.DisplayMetadata;
-using Icon = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.Icon;
 using ImplementationMetadata = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.ImplementationMetadata;
-using static MorganStanley.ComposeUI.Fdc3.DesktopAgent.Tests.TestData.TestAppDirectoryData;
-using System.Runtime.CompilerServices;
 
 namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Tests;
 
 public class EndToEndTests : IAsyncLifetime
 {
     private const string TestChannel = "fdc3.channel.1";
-    private readonly List<IModuleInstance> _runningApps = new();
+    private readonly List<IModuleInstance> _runningApps = [];
     private readonly object _runningAppsLock = new();
     private readonly ChannelTopics _topics = Fdc3Topic.UserChannel(TestChannel);
     private readonly Uri _webSocketUri = new("ws://localhost:7098/ws");
     private ServiceProvider _clientServices;
 
     private int _counter;
     private IHost _host;
-    private IMessageRouter _messageRouter;
+    private IMessaging _messageRouter;
```

Comment:
nitpick rename fields where it the message router is not applicable anymore (everywhere!)